### ISSUE TITLE
Add `mapActions()` helper to Pathify

### ISSUE THREAD

**Issue by davestewart** [davestewart/vuex-pathify#5]
It would actually be pretty useful to have a `mapActions()` style helper in Pathify.

Syntax to be decided, but along the lines of:

```js
methods: mapActions('products', [
  'load',
  'update'
])
```
```js
methods: mapActions({
  'loadProducts': 'products/load'
  'updateSettings': 'settings/update'
])
```
```js
methods: mapActions('products/load', 'settings/update', ...)
```
```js
methods: mapActions('products/*')
```

**Comment by mxlnk** [comment]
How do you feel about a mapMutations helper with the same interface? I'm currently struggling to incorporate my custom mutations into my components nicely.

**Comment by davestewart** [comment]
Maybe I don't understand what you're asking, but a map mutations shouldn't be needed as you have get and sync.

Can you give me more detail so I can help you out?

If you think it's a separate question, by all means open a new issue.

**Comment by davestewart** [comment]
Closed by b434fe06a79e08369a817c120e5c90af8ef41b0f
